etc/notes/CHANGES_5.0.0.md

```diff
@@ -16,6 +16,30 @@ The following is a detailed collection of the changes in the major v5 release of
 
 ## Changes
 
+### `Collection.mapReduce()` helper removed
+
+The `mapReduce` helper has been removed from the `Collection` class.  The `mapReduce` operation has been
+deprecated in favor of the aggregation pipeline since MongoDB server version 5.0.  It is recommended
+to migrate code that uses `Collection.mapReduce` to use the aggregation pipeline.
+
+If the `mapReduce` command must be used, the `Db.command()` helper can be used to run the raw
+`mapReduce` command.
+
+```typescript
+const command = {
```

Comment:
Should we show an example of how this would have looked with the helper syntax? Like for someone new to constructing a command themselves it's not clear that the mapReduce property is the collection name.

Also since the preferred migration is to use agg, can we leave a link to this page: https://www.mongodb.com/docs/manual/reference/map-reduce-to-aggregation-pipeline/

Reply:
done!